### DISCUSSION TITLE
運動改造大腦

### DISCUSSION THREAD

**Discussion by thz0000** [1101-GS4719-C/Python-Community-GS4719-C#1896] [General]
本學期的最後一篇，讓我用一門大二體育課的心得做結尾XD
-
要活就要動，運動是人的一生中不可或缺的一部分，連走路也是在運動，只是強度不大罷了。

雖然可能有些人不太喜歡運動，但運動的益處比我們想像的多更多。不只是身體素質，就連心理狀態、大腦運作都與運動息息相關，良好的運動習慣可以改造大腦、加強心理素質，像是穩定情緒、注意力集中、成績進步、增加社交等。

據研究，運動跟思考是使用同一組腦部迴路的，所以在強身健體的同時，學習速率也在變快，達到事半功倍的效果。此外，腦部結構和功能改變，誘發神經新生，還可以擁有預防大腦退化的成效。所以運動不僅是在訓練身體，更是在改造大腦。

心理層面，適當的壓力可以刺激大腦生長，過多也可能拖垮大腦，長期的壓力可能導致死亡，憂鬱就是在破壞連結，運動則負責重建連結。根據研究，治療心理疾病最好的方式是藥物結合運動，藥物可以提供立即的安全感，而運動可以從根本上解決焦慮問題。因為運動除了可以控制壓力產生的心理和生理影響，還能在細胞層面上發揮效果，達到從根本解決問題。

現在我們知道運動可以建立自信湧現活力、改造大腦，但運動的好處遠不止於此。運動還可以改善女性停經的不適症狀、產後憂鬱，可以刺激身體製造更多的胰島素受器，提高能量使用，再加上配合飲食與有氧、中度、激烈運動，健全的大腦與身心絕不是問題。

所以，如果想活到老並擁有良好的身心素質與大腦狀態，那就多運動吧！

**Comment by yuchien-lin** [comment]
我在小學時加入田徑隊的時候，教練也曾告訴我們這件事，運動對於我們大腦，心理都會有一定程度的影響，這也是我那時候待在田徑隊的原因，不過後來膝蓋受了重傷，就只能退隊了。高中開始，我也確實發現在情緒低落的時候，跑步能夠讓我心情平復，所以在那之後我心情不好的時候，我都會去跑步，我覺得非常有效。

**Comment by chuanmin0317** [comment]
在開始運動之後，我發現之前很多身體的情況有變好的趨向，像是我覺得自己的皮膚比起之前來的好很多，還有之前動不動就想睡覺，但在運動的習慣之後，就很少出現這種情況了，而且在運動完之後我發現我的集中力在一段時間內有所提升

**Comment by YifenLin** [comment]
我也很喜歡運動，雖然自已的運動神經不是很好，不過我很喜歡每次運動時進步的感覺，不過自己沒進步或是做不好的時候也會感到挫折，而這時候我會告訴自己現在就是磨練心志的好時機XD。

**Comment by benbebobo149** [comment]
我自己的經歷跟樓上的一樣，運動之後，精神就變超好，比較不容易睡著，上課也比較專注。除此之外，我覺得運動後會讓我的心情比較平靜一下，當我可能在床上Emo的時候，都會跑去操場跑個步轉換心情。運動不僅對身體有益，還能夠幫助我調整情緒，讓我在情緒起伏時找到一個出口。我也發現運動後的自己更有自信和積極的態度，這對我來說是一個巨大的改變。因此，我非常感謝這篇文章提供了關於運動的正面影響，這讓我更加堅定了持續運動的決心。最後，我要感謝同學的分享，這讓我意識到運動對身心健康的重要性。

**Comment by willychiuchiu** [comment]
每次只要運動完，真的可以明顯感受到大腦被活絡的感覺，尤其喜歡在夜晚運動，運動完流完汗再洗個澡，通體舒暢，這種感覺真的超級讚，也更好入睡，睡眠品質會有顯著的提升。運動的重要性我想已經被說到爛了，對於從小就有在接觸運動的我，身體狀況真的明顯比其他較少運動的同儕好上許多，不止體力較好，抵抗力也更好，較少生病感冒等，因此人真的活著就要動啊！

**Comment by YANRONG0123** [comment]
這個標題我常常在學校的某一位老師口中提到，而且我還修了他兩門體育課，有一說一，在過去幾學期養成定期慢跑之後，會感覺到身心靈舒暢，也不會浪費時間在無意義的事情上，我認為要保持長遠的運動，才能夠活絡我們的大腦，使我們在執行事情上更有效率。
